pcsx2/gui/AppInit.cpp

```diff
@@ -77,19 +72,6 @@ void Pcsx2App::OpenMainFrame()
 	DisassemblyDialog* disassembly = new DisassemblyDialog( mainFrame );
 	m_id_Disassembler = disassembly->GetId();
 
-#ifndef DISABLE_RECORDING
```

Comment:
Seems like the new location for initializing the recording windows is now a side-effect of the menubar being created?  Not sure how I feel about that, that feels unexpected and should probably remain here where other frames (like the debugger/console) are made.